internal/app/kube_helpers.go

```diff
@@ -13,8 +13,14 @@ import (
 // addNamespaces creates a set of namespaces in your k8s cluster.
 // If a namespace with the same name exists, it will skip it.
 // If --ns-override flag is used, it only creates the provided namespace in that flag
-func addNamespaces(namespaces map[string]namespace) {
+func addNamespaces(s state) {
```

Comment:
Since we use pointers to `state` everywhere perhaps we should use a pointer here as well.

Reply:
good catch, thanks :-) done